src/sidebar/components/annotation-header.js

```diff
@@ -24,6 +24,9 @@ function AnnotationHeader({
 }) {
   const annotationLink = annotation.links ? annotation.links.html : '';
   const replyPluralized = !replyCount || replyCount > 1 ? 'replies' : 'reply';
+  // NB: `created` and `updated` are strings, not `Date`s
```

Comment:
This comment may be unwarranted, but I was worried folks might get tripped over the equality statement otherwise as two `Dates` would be separate references.